identity-service/src/routes/userPlaylistUpdates.js

```diff
@@ -0,0 +1,90 @@
+const moment = require('moment-timezone')
+const { handleResponse, successResponse, errorResponseBadRequest, errorResponseServerError } = require('../apiHelpers')
+const models = require('../models')
+const authMiddleware = require('../authMiddleware')
+
+module.exports = function (app) {
+  // maybe we don't need this endpoint
+  /**
+   * Updates the lastPlaylistViews field for the user in the UserEvents table
+   * @param {string} walletAddress   user wallet address
+   */
+  app.get('/user_playlist_updates', handleResponse(async (req) => {
+    const { walletAddress } = req.query
+    if (!walletAddress) {
+      return errorResponseBadRequest('Please provide a wallet address')
+    }
+
+    try {
+      const userEvents = await models.UserEvents.findOne({
+        attributes: ['playlistUpdates'],
+        where: { walletAddress },
+        raw: true
+      })
+      if (!userEvents) throw new Error(`UserEvents for ${walletAddress} not found`)
+
+      return successResponse(userEvents.playlistUpdates)
+    } catch (e) {
+      req.logger.error(e)
+      // no-op. No user events.
+      return errorResponseServerError(
+        `Unable to get user last playlist views for ${walletAddress}`
+      )
+    }
+  }))
+
+  /**
+   * Updates the lastPlaylistViews field for the user in the UserEvents table
+   * @param {string} walletAddress            user wallet address
```

Comment:
walletAddy shouldn't be a param, you can actually pull it directly off of the authMiddleware

const { walletAddress } = req.user